iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/serviceclient/ExportImportTests.java

```diff
@@ -320,8 +344,8 @@ private static void runImportJob(List<ExportImportDevice> devices, ImportMode im
         // Creating the Azure storage blob and uploading the serialized string of devices
         InputStream stream = new ByteArrayInputStream(blobToImport);
         String importBlobName = "devices.txt";
-        CloudBlockBlob importBlob = importContainer.getBlockBlobReference(importBlobName);
-        importBlob.deleteIfExists();
+        BlobClient importBlob = importContainer.getBlobClient(importBlobName);
```

Comment:
isn't this supposed to be `BlockBlobClient` ?

Reply:
Bob Lawblaw's Block Blob

https://arresteddevelopment.fandom.com/wiki/Bob_Loblaw

Reply:
Good catch, it should be. I'll fix it